_posts/homepage/2018-04-12-Sprints.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+image: stickers.jpg
+title: Sprints
+description: PyData Ams
+category: [homepage]
+---
+
+For the first time in its' history, PyData Amsterdam will host Open Source Contribution Sprints. These will take place
+on Monday the 15th.
+
+Our sprints are a day of intensive learning and development on an open source project of your choice.
+It’s a time to come together, to share what you’ve learned during the conference with the people you met, 
+and apply it to an open source project.
+
+We will bring together eager developers and maintainers of some of the libraries we all use and love to learn and 
+do our share in helping out with open source development. 
+ 
+
+# Who can participate?
+You! All experience levels are welcome; sprints are a great opportunity to get connected with, 
+and start contributing to your favorite Python project. Participation in the sprints is free! We do ask you to
+sign up so we can make sure we have enough facilities in place to host everyone.
```

Comment:
We can already set up a 'sign up' form which we link to on this page?

Reply:
are we doing that in eventbrite or in google forms?

Reply:
I think we'd want to do an 'I'm interested in learning more about sprints and possibly participating' checkbox on the Eventbrite page so we have an idea of how many people are interested, and only open up the real registration for the sprints fairly close to the conference (when people know for sure whether they're able to make it).